modules/examples/health-check/health-server-monix/hc-server.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file seems to be related to your IDE. Could we remove it and ignore it from the .gitignore file?

Reply:
Sure, no problem.  I can't understand why it didn't happen before. 

What should I do with code coverage test? It fails